templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -444,20 +444,28 @@ contents:
       # Then activate all the connections
       for conn in "${connections[@]}"; do
         local active_state=$(nmcli -g GENERAL.STATE conn show "$conn")
-        if [ "$active_state" != "activated" ]; then
-          for i in {1..10}; do
-            echo "Attempt $i to bring up connection $conn"
-            nmcli conn up "$conn" && s=0 && break || s=$?
-            sleep 5
-          done
+        # Connection already active, only set autoconnect
+        if [ "$active_state" == "activated" ]; then
```

Comment:
Note: I did this here + the continue to make this a bit more readable (at the cost of having 2 lines with nmcli ... connection.autoconnect yes). The if/else became a bit difficult to read otherwise.